examples/co2brinepvt.cpp

```diff
@@ -160,15 +169,15 @@ int main(int argc, char **argv)
         }
     } else if (prop == "internalEnergy") {
         if (phase == "CO2") {
-            value = co2Pvt.internalEnergy(/*regionIdx=*/0 ,T,p, rv);
+            value = co2Pvt.internalEnergy(/*regionIdx=*/0 ,T,p, rv, rv);
         } else if (phase == "brine") {
             value = brineCo2Pvt.internalEnergy(/*regionIdx=*/0 ,T,p, rs);
         } else {
             throw std::runtime_error("phase " + phase + " not recognized. Use either CO2 or brine");
         }
     } else if (prop == "enthalpy") {
         if (phase == "CO2") {
-            value = p / densityGas(co2Pvt, p, T, rv) + co2Pvt.internalEnergy(/*regionIdx=*/0 ,T,p, rv);
+            value = p / densityGas(co2Pvt, p, T, rv) + co2Pvt.internalEnergy(/*regionIdx=*/0 ,T,p, rv, rv);
```

Comment:
Here you pass same values rv and rv, and inside in Co2GasPvt.hpp you check which one is greater `const Evaluation xBrine = convertRvwToXgW_(max(rvw,rv),regionIdx);` . I see similar constructions for other properties and function calls. Is it intended to be changed at some point? I suggest for now adding extra comments to explain the use of oil to represent brine to help out when in the future we try to expand CO2STORE for depleted fields.

Reply:
Thanks for the comment. I have added an assert and know pass either rv = 0 or rvw = 0.